code/backend/batch/utilities/helpers/AzureSearchHelper.py

```diff
@@ -31,9 +31,7 @@ def get_vector_store(self):
                 name="content_vector",
                 type=SearchFieldDataType.Collection(SearchFieldDataType.Single),
                 searchable=True,
-                vector_search_dimensions=len(
-                    llm_helper.get_embedding_model().embed_query("Text")
-                ),
+                vector_search_dimensions=int(env_helper.AZURE_SEARCH_DIMENSIONS),
```

Comment:
Can we leave a link here to where people will find what this value should be?
https://learn.microsoft.com/en-us/azure/search/vector-search-index-size?tabs=rest-vector-quota#memory-overhead-from-the-selected-algorithm

(and maybe in the EnvHelper)

Reply:
Wouldn't the README make more sense?

Also I think it would be better to link to this: https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#embeddings-models

Reply:
Yes - much better! :-)